VideoWeb/VideoWeb.Contract/Responses/VideoEndpointResponse.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using VideoWeb.Common.Models;
+
+namespace VideoWeb.Contract.Responses
+{
+    public class VideoEndpointResponse
+    {
+        /// <summary>
+        /// The endpoint id
+        /// </summary>
+        public Guid Id { get; set; }
+
+        /// <summary>
+        /// The endpoint display name
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// The endpoint sip address
+        /// </summary>
+        public string SipAddress { get; set; }
```

Comment:
I don't think we should expose the SIP address or PIN to the FE. These details will not be used by the app and thus should not be made public to the app

Reply:
removed